app/commands/user/update_roles.rb

```diff
@@ -0,0 +1,10 @@
+class User::UpdateRoles
+  include Mandate
+
+  initialize_with :user, :roles
```

Comment:
How about we do it like this instead, and then encapsulate the actual manipulation within this command?

```suggestion
  initialize_with :user, add: [], remove: []
```

Reply:
Maybe with `remove_all: false` or something to handle the reset_account scenario?

Reply:
Do three commands